lib/YaST/SystemSettings/SystemSettingsPage.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for YaST module
+# System Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::SystemSettings::SystemSettingsPage;
+use strict;
+use warnings;
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{menu_btn_add} = $self->{app}->menucollection({label => 'Add...'});
```

Comment:
I combined it with the PCI ID Setup Tab, but I could separate it.

Reply:
it would be better, both are doing the same and I understand your approach, the problem potentially could be with those same buttons in both and the transitions. But still is ok, not asking to change it if it works.